docs/installation.md

```diff
@@ -87,6 +87,7 @@ SMACK depends on the following projects:
 * [Mono][] version 5.0.0 or greater (except on Windows)
 * [Z3][] or compatible SMT-format theorem prover
 * [Boogie][] or [Corral][] or compatible Boogie-format verifier
+* [sea-dsa][]
```

Comment:
Shouldn’t we add Boost to the list of requirements, since Smack code now uses it directly?

Reply:
Boogie is a dependency of sea-dsa. I think we should not list secondary dependencies.

Reply:
I would agree if you were consuming sea-dsa as an already-built library, but you are instead building sea-dsa directly in the build of Smack. This means that Smack’s build requires Boost.

If building sea-dsa were separate from building Smack, then I would agree that Boost could be considered not a direct dependency of Smack.

Reply:
Sounds good. Let me put Boost in.